sre-recipes/recipes/configs_based/README.md

```diff
@@ -0,0 +1,128 @@
+# Configs-Based SRE Recipe
+
+This directory contains configs for supported config-based SRE Recipes.
+
+## Writing the Config
+
+Each SRE Recipe config is a YAML file with the following overall structure.
+
+```
+name: <Put the name of your recipe here. This is not user visible.>
+description: <Put the description of your recipe here. This is not user visible.>
+config:
+  # This section lists the configs for all the actions to perform for this SRE
+  # Recipe. In practice, this is where you try to break sandbox services.
```

Comment:
I noticed this sentence is in both break and restore:

"This section lists the configs for all the actions to perform for this SRE Recipe."

I think it could be a bit more clear. Maybe: 

"This section defines the actions that are run when `sandboxctl sre-receipes break` is called"?

Reply:
Done.